### PR TITLE
feat: Add scripts to separate e2e tests for chectl

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: flacatus <flacatus@redhat.com>

### What does this PR do?
This PR add ability to separate e2e tests by platform(minikube and minishift). More info in Ref. issue.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16932

### CODE REVIEW

File: .ci/cico_chectl_minikube.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+#   Red Hat, Inc. - initial API and implementation
+
+set -e -x
+
+#Stop execution on any error
+trap "fail_trap" EXIT
+
+init() {
+  SCRIPT=$(readlink -f "$0")
+  SCRIPTPATH=$(dirname "$SCRIPT")
```
Comment (AndrienkoAleksandr) [inline .ci/cico_chectl_minikube.sh:20]: You are using undescore everywhere, so maybe SCRIPTPATH => SCRIPT_PATH . And name of this variable for me associated with full path to the script, maybe name SCRIPT_DIR will be better?

Comment (flacatus) [reply .ci/cico_chectl_minikube.sh:20]: ok, I'll change.

Comment (flacatus) [reply .ci/cico_chectl_minikube.sh:20]: Sound correct for me.

**Comment by flacatus** [comment]
We skip `e2e_tests` job because will be removed after this merge.

**Comment by flacatus** [comment]
[test]

**Pull request merged**